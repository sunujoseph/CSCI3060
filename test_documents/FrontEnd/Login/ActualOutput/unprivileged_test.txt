Enter Username: 
Error: You Do Not Have Privileges To Perform This Transaction
Error: You Do Not Have Privileges To Perform This Transaction
Error: You Do Not Have Privileges To Perform This Transaction
