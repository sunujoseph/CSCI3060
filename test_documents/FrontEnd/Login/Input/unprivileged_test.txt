login
userBS
create
delete
refund
