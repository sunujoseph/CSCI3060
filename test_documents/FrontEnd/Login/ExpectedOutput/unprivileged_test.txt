Enter Username: 
Error: Privileged transactions are restricted to admins only
Error: Privileged transactions are restricted to admins only
Error: Privileged transactions are restricted to admins only